cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java

```diff
@@ -17,101 +17,95 @@
-            DEFAULT_MAX_UNCOMPRESSED_TEST_EVENT_SIZE = 5 * 1024,
-            DEFAULT_SESSIONS_CACHE_SIZE = 100,
-            DEFAULT_SCOPES_CACHE_SIZE = 10,
-            DEFAULT_PAGE_SESSION_CACHE_SIZE = 100,
-            DEFAULT_PAGE_SCOPES_CACHE_SIZE = 100,
-            DEFAULT_SESSION_STATISTICS_CACHE_SIZE = 10_000,
-            DEFAULT_GROUPS_CACHE_SIZE = 10_000,
-            DEFAULT_EVENT_BATCH_DURATION_CACHE_SIZE = 5_000,
-            DEFAULT_PAGE_GROUPS_CACHE_SIZE = 10_000,
-            DEFAULT_COUNTER_PERSISTENCE_INTERVAL_MS = 1000;
-	public static final long DEFAULT_BOOK_REFRESH_INTERVAL_MILLIS = 60000,
-            DEFAULT_EVENT_BATCH_DURATION_MILLIS = 5_000;
-
-
-
-    private final NetworkTopologyStrategy networkTopologyStrategy;
-    private final long timeout;
-    private final ConsistencyLevel writeConsistencyLevel,
-            readConsistencyLevel;
-    private String keyspace,
-            schemaVersion;
-    private int keyspaceReplicationFactor;
-
-    private int maxParallelQueries, //FIXME: remove
-            resultPageSize,
-            maxMessageBatchSize,
-            maxUncompressedMessageBatchSize,
-            maxTestEventBatchSize,
-            maxUncompressedTestEventSize,
-            sessionsCacheSize,
-            scopesCacheSize,
-            pageSessionsCacheSize,
-            pageScopesCacheSize,
-            sessionStatisticsCacheSize,
-            pageGroupsCacheSize,
-            groupsCacheSize,
-            eventBatchDurationCacheSize,
-            counterPersistenceInterval,
-            composingServiceThreads;
-
-    private SelectExecutionPolicy multiRowResultExecutionPolicy, singleRowResultExecutionPolicy;
-
-	private long bookRefreshIntervalMillis, eventBatchDurationMillis;
-
-    public CassandraStorageSettings() {
-        this(null, DEFAULT_TIMEOUT, DEFAULT_CONSISTENCY_LEVEL, DEFAULT_CONSISTENCY_LEVEL);
-    }
+    public static final int DEFAULT_KEYSPACE_REPL_FACTOR = 1;
+    public static final int DEFAULT_MAX_PARALLEL_QUERIES = 500;
+    public static final int DEFAULT_RESULT_PAGE_SIZE = 0;
+    public static final int DEFAULT_MAX_UNCOMPRESSED_MESSAGE_BATCH_SIZE = 5 * 1024;
+    public static final int DEFAULT_MAX_UNCOMPRESSED_TEST_EVENT_SIZE = 5 * 1024;
+    public static final int DEFAULT_SESSIONS_CACHE_SIZE = 100;
+    public static final int DEFAULT_SCOPES_CACHE_SIZE = 10;
+    public static final int DEFAULT_PAGE_SESSION_CACHE_SIZE = 100;
+    public static final int DEFAULT_PAGE_SCOPES_CACHE_SIZE = 100;
+    public static final int DEFAULT_SESSION_STATISTICS_CACHE_SIZE = 10_000;
+    public static final int DEFAULT_GROUPS_CACHE_SIZE = 10_000;
+    public static final int DEFAULT_EVENT_BATCH_DURATION_CACHE_SIZE = 5_000;
+    public static final int DEFAULT_PAGE_GROUPS_CACHE_SIZE = 10_000;
+    public static final int DEFAULT_COUNTER_PERSISTENCE_INTERVAL_MS = 1000;
+	public static final long DEFAULT_BOOK_REFRESH_INTERVAL_MILLIS = 60000;

```

Comment:
Let's correct the format for that line as well. We already changed all lines here, so why leave this one?